stdlib/tkinter/__init__.pyi

```diff
@@ -666,7 +666,7 @@ class Wm:
     iconmask = wm_iconmask
     def wm_iconname(self, newName: Incomplete | None = None) -> str: ...
     iconname = wm_iconname
-    def wm_iconphoto(self, default: bool, __image1: Image, *args: Image) -> None: ...
+    def wm_iconphoto(self, default: bool, __image1: _ImageSpec, *args: _ImageSpec) -> None: ...
```

Comment:
Ideally this wouldn't take `_ImageSpec` because it does not work with bitmap images. But I don't have any better suggestions. Tkinter images have a `.type()` method that returns `'photo'` or `'bitmap'`, but it doesn't help because PIL images don't have it.

Reply:
Perhaps instead of a protocol, we could replace `_Image` with a pair of typing-only marker abstract base classes, that the tkinter and PIL images inherit from.